src/AsmResolver.PE/DotNet/Builder/DotNetSegmentBuffer.cs

```diff
@@ -30,6 +27,11 @@ public DotNetSegmentBuffer(IDotNetDirectory dotNetDirectory)
             AddIfPresent(dotNetDirectory.DotNetResources);
             AddIfPresent(dotNetDirectory.StrongName);
             AddIfPresent(dotNetDirectory.VTableFixups);
+            if (dotNetDirectory.VTableFixups?.Count > 0)
+            {
+                var buffer = new VTableEntriesBuffer(dotNetDirectory.VTableFixups);
+                AddIfPresent(buffer);
```

Comment:
Unnecessary AddIfPresent call. `buffer` is always non-null. We can just use `_segments.Add` here.